Fix tests

davidyuk opened the pull request:
https://github.com/facebook/flow/issues/6049

davidyuk committed:
Update package scripts and Travis config

davidyuk committed:
Fix tests and run prettier

edwarddikgale reviewed (changes requested):
Denis the line to run test code in package.json is removed: "test-node": "cd packages/test && yarn run test-node" so tests won't run.

And also there is a hard coded I think address: ae.signTransaction(0, "e8018504e3b292008252089428ee52a8f3d6e5d15f8b131996950d7f296c7952872bd72a2487400080") in Ae.js, can you put this as a variable somewhere so its defined and easy to change thereof...

davidyuk commented:
@edwarddikgale thanks for the review!

> the line to run test code in package.json is removed: "test-node"

This package is a fork of `LedgerHQ/ledgerjs` repository, that repository has a complex structure, it contains several NPM packages implementing APIs for different currencies and transports. In e0c844e9c2e5a6b77aaa64f97a21382f5f993640 I have removed all packages except `hw-app-eth`, but I forgot to update some stuff in package.json. `test-node` and `test-browser` are referenced to package for snapshot testing that was also removed. Such tests can be useful, but they can be [runned only manually](https://github.com/LedgerHQ/ledgerjs/tree/master/packages/test#notes-on-tradeoffs). Should I bring snapshot testing back and write some tests for this package?

> And also there is a hard coded I think address

It is an example transaction, it is left as it was in the [original package](https://github.com/LedgerHQ/ledgerjs/blob/2b7cac197e79fd045d244d4a02b81d11f8d8579c/packages/hw-app-eth/src/Eth.js#L109). Also, it is a part of the comment, so I can't extract it into a variable, but we can write something else there, what do you prefer?

edwarddikgale commented:
Ok I got it. Thank you for elaborating on this. I am approving it
